src/pages/game.tsx

```diff
@@ -0,0 +1,13 @@
+import Head from 'next/head'
+import styles from '@/styles/Home.module.css'
+
+export default function Menu() {
+  return (
+    <>
```

Comment:
No need to have the imports or the empty tag in here, the whole file can just be:

```tsx
export default function Menu() {
  return (
    <main>
      <h1>This is the GAME</h1>
    </main>
  );
}
```